README.md

```diff
@@ -1 +1,27 @@
-# hw01
\ No newline at end of file
+# hw01
+by Tina Chen
+
+### Biography
+Hi! my name is *Peiying Chen*, but I go by **Tina**. I am a first year in the college double majoring in **economics and psychology**. I came from China, and I moved to Irvine, California when I was in middle school. 
+
+During my free time, I enjoy doing: 
+
+* exercising 
+  * yoga
+  * swimming 
+* embroidery
+* binge watching
+  * some of my favorite tv shows are: 
+    1. Criminal minds
+    1. Pysch
+    1. Selling Sunset
+    1. 90 Days Fiance (*my guilty pleasure*)
```

Comment:
A non-computing suggestion is to not feel guilty about things you find pleasure in haha! But nice use of italics also:) Perhaps linking these TV shows would have been a nice touch if someone would like to know where to watch them.

Reply:
Haha perhaps you could use the strikethrough over the words "my guilty pleasure" using ~~ on either side of the words, that could have added more to the humor:)